src/routes/u/[playerId].svelte

```diff
@@ -139,6 +139,15 @@
       await fetcher(`/api/user/${player.id}/giveRole/${role}`, { withCredentials: true });
       refreshPlayerData({ forceRevalidate: true, softRefresh: true });
    }
+
+   let isOculusPlayer : boolean;
+   $: isOculusPlayer = !!$playerData?.profilePicture?.endsWith('oculus.png');
+
+   let playerId : string | null;
+   $: playerId = $playerData?.id ?? null;
+
+   let destination : string | null;
+   $: destination = playerId && !isOculusPlayer ? `https://steamcommunity.com/profiles/${playerId}` : null
```

Comment:
I'd prefer to keep this kind of stuff out of the script tags and do these kind of inline statements in the html

Reply:
Svelte way is that markup is as simple as possible, and all processing or destructuring as in this case is done in script. But of course I'll move it to the markup if you wish so.

Reply:
What coding style should I adopt then? The Svelte way is declarative rather than imperative. For example, this snippet: 

https://github.com/ScoreSaber/ScoreSaber-Frontend/blob/8bbb9f446d5010d1d5e04507acbdb01a43b1695e/src/routes/u/%5BplayerId%5D.svelte#L68-L76

should look more like this:
```js
$: refreshScores({newUrl: getPlayerScoresUrl($page.params.playerId, $page.query.toString())});
$: refreshPlayerData({ newUrl: getPlayerInfoUrl($page.params.playerId) });
```

You declare dependencies and let Svelte do the rest.

Reply:
I moved the playerId and destination back to markup and changed the way I check the Steam player.

Reply:
I just resolved the conflicts with the current main branch. Do you want me to change anything else in this PR @Umbranoxio ?